knowhere/index/vector_index/IndexAnnoy.cpp

```diff
@@ -81,6 +82,7 @@ IndexAnnoy::Load(const BinarySet& index_binary) {
 
 void
 IndexAnnoy::BuildAll(const DatasetPtr& dataset_ptr, const Config& config) {
+    utils::SetBuildOmpThread(config);
```

Comment:
this setting should be set in line 91, after this check:
    if (index_) {
        // it is builded all
        LOG_KNOWHERE_DEBUG_ << "IndexAnnoy::BuildAll: index_ has been built!";
        return;
    }

Reply:
ok